system/default_ad_api_helpers/src/routing_adaptor.hpp

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 Autoware Foundation
```

Comment:
Same here.

Reply:
The original code is part of [this file](https://github.com/autowarefoundation/autoware.universe/blob/main/planning/mission_planner/include/mission_planner/mission_planner_base.hpp).

Reply:
Since it's quite different from the original file, I think you don't necessarily have to use the original license.

Reply:
Done https://github.com/autowarefoundation/autoware.universe/pull/1494/commits/7eda408a79b0ed4e8f69d987c8cf413deeabd93c.